docker/images/outrigger/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:14.04
```

Comment:
A Mac user could build the bits inside a container...

Seriously this seems consistent with how other docker images that I've seen work. For example the postgres container doesn't include the build steps and obviously if you downloaded the postgres source and built it on a Mac it wouldn't be suitable as the basis for a container image. I think the same thing applies here.